src/pages/_app.tsx

```diff
@@ -7,7 +7,7 @@ export default function App({ Component, pageProps }: AppProps) {
     <>
       <Head>
         <title>FilmFinder</title>
-        <meta name="description" content="Generated by create next app" />
+        <meta name="description" content="Find films, and bookmark them to review later." />
```

Comment:
It'd be nice if it was possible to set this once, at the time of an app's creation, and this text automatically appeared in

1. the `About` section of the GitHub repository,
2. the `description` meta tag, and maybe
3. the `description` field in `package.json`